packages/react/src/section-alert/SectionAlert.tsx

```diff
@@ -58,34 +60,50 @@ export const SectionAlert = forwardRef<HTMLDivElement, SectionAlertProps>(
 			focusOnUpdate,
 			forwardedRef,
 		});
+		const { childrenId, titleId, toneId } = useSectionAlertIds(id);
 
-		const Icon = sectionAlertIconMap[tone];
+		const icon = sectionAlertIconMap[tone];
 		const closeHandler = getOptionalCloseHandler(onClose, onDismiss);
 
 		return (
 			<Flex
+				{...props}
 				alignItems="center"
+				aria-labelledby={`${toneId} ${titleId} ${children ? childrenId : ''}`}
 				background={tone}
 				borderColor={tone}
 				borderLeft
 				borderLeftWidth="xl"
+				focusRingFor="all"
 				gap={0.5}
 				highContrastOutline
 				id={id}
-				focusRingFor="all"
 				justifyContent="space-between"
 				padding={1}
 				ref={ref}
-				role={role}
+				// Not using default arg because is if someone accidentally passes a falsey value, we still need the role to be set.
+				role={role || 'region'}
 				rounded
 				tabIndex={tabIndex ?? (focusOnMount || focusOnUpdate ? -1 : undefined)}
-				{...props}
 			>
 				<Flex gap={0.5}>
-					{Icon}
-					<Flex gap={0.25} flexDirection={'column'}>
-						{title && <Text fontWeight="bold">{title}</Text>}
-						{children}
+					<span
+						css={{
+							display: 'inline-flex',
+						}}
+					>
+						{icon}
+						<span css={visuallyHiddenStyles} id={toneId}>
```

Comment:
Wanted to use `aria-label`, but there's no way of using it on a generic element that axe and ARC won't complain at. This means that iOS's virtual cursor is the size of the word and not the icon, but it's better than any alternative